jans-cedarling/cedarling/src/authz/request.rs

```diff
@@ -13,11 +13,11 @@ use cedar_policy::{EntityId, EntityTypeName, EntityUid, ParseErrors};
 #[derive(Debug, Clone, serde::Deserialize)]
 pub struct Request {
     /// Access token raw value
-    pub access_token: String,
+    pub access_token: Option<String>,
```

Comment:
I don't see implementation of [this](https://github.com/JanssenProject/jans/issues/10410#issuecomment-2539607424) in current PR

Reply:
because there's another [issue](https://github.com/JanssenProject/jans/issues/10437) for that.